pennylane/labs/resource_estimation/ops/op_math/symbolic.py

```diff
@@ -30,12 +30,159 @@
 
 
 class ResourceAdjoint(AdjointOperation, re.ResourceOperator):
-    """Resource class for the Adjoint symbolic operation."""
+    r"""Resource class for the symbolic AdjointOperation.
+
+    A symbolic class used to represent the adjoint of some base operation.
+
+    Args:
+        base_class (Type[~.ResourceOperator]): the class type of the base operator that is adjointed
```

Comment:
```suggestion
        base_class (Type[~.ResourceOperator]): the class type of the base operator that is adjointed
```

Maybe there's a better way to say "adjointed". The class type of the base operator that we want the adjoint of.

Reply:
Fixed 👍🏼